packages/attendance/package.json

```diff
@@ -87,6 +87,7 @@
     "@storybook/preset-create-react-app": "^4.0.1",
     "@storybook/react": "^6.4.19",
     "@storybook/testing-library": "^0.0.9",
+    "@types/react-native": "^0.69.3",
```

Comment:
@coolbung , Do we need react native dependency?

@sidtohan - where is dependency used?

Reply:
No we don't. Any reason to have it @sidtohan ?

cc @sagarkoshti1990

Reply:
@coolbung @arajput 
The main reason why I added is to ensure that typescript is able to identify some of props that exist on many of the react native web components. For eg, typescript was unable to identify the ```onPress``` prop that exists on some react native web components. But installing this seems to provide the typescript compiler with the information that the said prop exists on the given component.
I'm currently looking for some other fixes as well, but so far this seems to work.